aict_tools/io.py

```diff
@@ -164,10 +238,24 @@ def __init__(
         self.aict_config = aict_config
         self.columns = columns
         self.feature_generation_config = feature_generation_config
-        if aict_config.has_multiple_telescopes:
-            self.n_rows = get_number_of_rows_in_table(path, aict_config.array_events_key)
+        if aict_config.data_format == "CTA":
+            with tables.open_file(path) as t:
+                if aict_config.telescopes:
+                    self.tables = [
+                        (f"/dl1/event/telescope/parameters/{tel.name}", len(tel))
+                        for tel in t.root.dl1.event.telescope.parameters
+                        if tel.name in aict_config.telescopes
```

Comment:
This forces people to write `telescopes: [tel_001, tel_002, ...]`, right? Shouldn't we allow just `[1, 2, 3, ...]`?

Reply:
Yes it does.
I didn't really make up my mind on what I preferred and just started with this.